### PR TITLE
Updates to address part of bug 1230483

### PR DESCRIPTION
**Pull request by gashcrumb**
@liggitt FYI, though waiting on Allie about placement of the connect icons in the overview detail panel, as the latest design I've seen won't really work for multiple containers.  Think it's going to require updating kubernetes-object-describer to add in an extension.

### CODE REVIEW

File: assets/app/styles/_extensions.less [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.pod-template div[hawtio-extension] {
+  position: relative;
+}
+
+.pod-template .connect-link {
+  position: absolute;
+  right: -5px;
+  top: -66px;
```
Comment (liggitt) [inline assets/app/styles/_extensions.less]: where is this attempting to position the link?

File: assets/app/scripts/extensions/javaLink.js [hunk 2]
```diff
@@ -24,9 +24,9 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
   })
   .run(function(ProxyPod, BaseHref, HawtioExtension, $templateCache, $compile, AuthService) {
     var template =
-      '<span ng-show="jolokiaUrl" title="View java details">' +
+      '<span class="connect-link" ng-show="jolokiaUrl" title="Connect to container">' +
       '  <a ng-click="gotoContainerView($event, container, jolokiaUrl)" ng-href="jolokiaUrl">' +
-      '    <i class="fa fa-external-link"></i>' +
+      '    <i class="fa fa-sign-in"></i>' +
```
Comment (liggitt) [inline assets/app/scripts/extensions/javaLink.js]: Wasn't there a request to add text beside the icon to make it clearer what it was?

File: assets/app/styles/_extensions.less (hunk 1 above)
Comment (gashcrumb) [inline assets/app/styles/_extensions.less]: Top-right corner of that box as per the current suggested design...

File: assets/app/scripts/extensions/javaLink.js (hunk 2 above)
Comment (gashcrumb) [inline assets/app/scripts/extensions/javaLink.js]: I know it was mentioned but the design doesn't have it, other than the 'title' attribute which I've changed to match the document.

File: assets/app/styles/_extensions.less (hunk 1 above)
Comment (liggitt) [inline assets/app/styles/_extensions.less]: ok... you can't assume these offsets will work. containers have variable amounts of stuff in them (some have associated builds, others don't, some have associated source commits, others don't). Additionally, these offsets don't take line wrapping into account.

Comment (gashcrumb) [inline assets/app/styles/_extensions.less]: Agreed, ideally the container-links extension would perhaps be before/after the 'name' in the template, then I can just right-justify it via 'right' and it'll be basically in the same spot.  I recall though being asked to put the extension in the template where it currently is, so it doesn't give me much choice but to work around it via css positioning.

File: assets/app/scripts/extensions/javaLink.js (hunk 2 above)
Comment (liggitt) [inline assets/app/scripts/extensions/javaLink.js]: We can iterate on appearance, but go ahead and add the text beside the icon... it's impenetrable as-is

File: assets/app/styles/_extensions.less (hunk 1 above)
Comment (liggitt) [inline assets/app/styles/_extensions.less]: Absolute positioning to the right would still allow overlapping content, even if we moved container-links to the top. I think we need to leave the link where it is for the moment

Comment (gashcrumb) [inline assets/app/styles/_extensions.less]: Agreed!  Have updated to reflect that, already looks a lot better with the text in place :-)

**Comment by liggitt** [comment]
LGTM, @smarterclayton merge

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**